internal/cli/atlas/deployments/search/indexes/create.go

```diff
@@ -127,7 +126,7 @@ func CreateBuilder() *cobra.Command {
 	opts := &CreateOpts{
 		IndexOpts: search.IndexOpts{
 			Analyzer: search.DefaultAnalyzer,
-			Dynamic:  false,
+			Dynamic:  true,
```

Comment:
both the docs https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Atlas-Search/operation/createAtlasSearchIndex and the existing [atlas search create cmd](https://github.com/mongodb/mongodb-atlas-cli/blob/a89fe8d8c25534b91fbb34e2bb7a7a5b5cc2ff69/internal/cli/atlas/search/create.go#L133) are using `false` as default value. You think we should change it to `true` here?

Reply:
basically because there are no fields to configure